edgecast/resources/originv3/origin_schema.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 Edgecast Inc., Licensed under the terms of the Apache 2.0 license.
+// See LICENSE file in project root for terms.
+package originv3
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func GetOriginV3Schema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
```

Comment:
We need a `platform` schema argument to determine which endpoint to call.

Reply:
N/A separate resources for adn/httplarge